### DISCUSSION TITLE
Welcome to geist-ui-svelte Discussions!

### DISCUSSION THREAD

**Discussion by ieedan** [ieedan/geist-ui-svelte#68] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ieedan** [comment]
@SikandarJODD I turned on discussions so we can use these

**Comment by SikandarJODD** [comment]
I can see website on Light Mode : 
![image](https://github.com/ieedan/geist-ui-svelte/assets/93428946/968386e2-84e4-4783-bdfd-aac54a17bc4f)
i think color should be black right ?

**Reply by ieedan** [reply]
No I believe this is intended kinda gray like it is on dark mode

**Reply by ieedan** [reply]
Darker than the section header but lighter than when selected

**Reply by SikandarJODD** [reply]
i think it should like more gray ..and when we select it should be blue normally when we select it's blue

**Reply by SikandarJODD** [reply]
![image](https://github.com/ieedan/geist-ui-svelte/assets/93428946/551dc0d8-3b72-4720-916d-f5e0a84bfc38)

**Reply by SikandarJODD** [reply]
Hey what things i should work on further ? any specific thing i should work on ?

**Reply by ieedan** [reply]
If you want to make those darker that works for me. Just make sure you make them lighter in dark mode as well.

**Reply by ieedan** [reply]
Also look at #72 if we want to upgrade to v3.4.1 or greater of tailwind there are some changes that need to be made.

Look at this issue https://github.com/tailwindlabs/tailwindcss/issues/12764

Basically they made a breaking change that will change the way we have to write our component code.

A possible fix is either to use the new `darkMode:` selector instead of `dark:` or to add `dark:` to any spots in the codebase where the class is the same for light and dark. I prefer the latter because we don't use the `darkMode` selector

**Reply by ieedan** [reply]
Basically as you make changes remember to write your tailwind in this new way so we don't have to correct it when we upgrade. I need to make sure I am doing the same thing.

**Reply by ieedan** [reply]
Right now I am going page by page through the docs and finding any bugs and just correcting them right then. Adding any potential features I can think of. 

I think it may also be good if one of us builds a simple project using the library to work out any pain-points we can find. I'm not sure if you have used it to build anything yet I certainly haven't had the chance.

**Reply by SikandarJODD** [reply]
i will start building a simple project using geist-ui-svelte

**Reply by ieedan** [reply]
Sounds good are you going to open a PR for the text color on the nav bar?

**Reply by SikandarJODD** [reply]
yes done

**Reply by ieedan** [reply]
If you can make it a public repo we can link to it as an example project. 

Any ideas on what you will build?

**Comment by SikandarJODD** [comment]
### Check out :  Light Mode
![image](https://github.com/ieedan/geist-ui-svelte/assets/93428946/4584147e-8da2-48ef-8a62-00ad40deac02)

### Dark Mode : only changes color='abort' to no color ....
![image](https://github.com/ieedan/geist-ui-svelte/assets/93428946/42bb623d-c194-4d60-a75a-6e5089fac479)

and it look good

**Reply by ieedan** [reply]
I like it.

 I think that also fixes a lighthouse issue with the contrast on the header?
